nicolalandro_ntsnet-cub200_ntsnet.md

```diff
@@ -0,0 +1,120 @@
+---
+layout: hub_detail
+background-class: hub-background
+body-class: hub
+category: researchers
+title: ntsnet
+summary: a fine grane model for image classification.
+image: nts-net.png
+author: Moreno Carraffini and Nicola Landro
+tags: [vision]
+github-link: https://github.com/nicolalandro/ntsnet-cub200/archive/master.zip
+featured_image_1: no-image
```

Comment:
it would be good to add an image of a bird, maybe from the dataset. it is helpful for users when they browse the https://pytorch.org/hub/ page

Reply:
Ok!